scripts/prepare_sector_network.py

```diff
@@ -1384,6 +1385,27 @@ def add_heat(network):
                              capital_cost=costs.at['micro CHP','fixed'],
                              lifetime=costs.at['micro CHP','lifetime'])
 
+    if options["geo_thermal"]:
+
+        try:
+            print("adding geothermal district heating in germany")
+            costs_and_potentials = pd.read_csv(
+            '../../geothermal_potential_germany.csv')
+            network.add("Carrier",name + " geo thermal")
+
+            for i, row in costs_and_potentials.iterrows():
+
+                network.add("Generator",
+                             f"DE0 0 urban central geo thermal {i}",
+                             bus="DE0 0 urban central heat",
+                             carrier="urban central geo thermal",
+                             p_nom_extendable=True,
+                             p_nom_max = row['potential [MW]'],
+                             capital_cost=annuity(30,0.07)*row['cost [EUR/kW]']*1e6,
```

Comment:
Yes, I choose the 7% because it was used globally. I will update this and use the one from the config file. 
There are different values for the lifetime also for other technologies in the technology-data repo. But I don't wanted to change the other repo, so I added the value manually. The lifetime for geothermal is taken from a source I also used to calculate the geothermal potential. I will add some documentation and cite the source here.

Reply:
I updated the discount rate and added a comment to reference the lifetime.